README.md

```diff
@@ -96,69 +96,6 @@ Version 1.0.0 of this module introduces a breaking change: adding the `disable-l
 In either case, upgrading to module version `v1.0.0` will trigger a recreation of all node pools in the cluster.
 
 [^]: (autogen_docs_start)
-
```

Comment:
Looks like the docs got lost somehow. Try `make generate_docs`.

Reply:
> Looks like the docs got lost somehow. Try make generate_docs.
Done!